pandas/core/frame.py

```diff
@@ -2786,7 +2794,7 @@ def info(
             show_counts=show_counts,
         )
 
-    def memory_usage(self, index=True, deep=False) -> Series:
```

Comment:
why is the return type removed?

Reply:
it shouldn't be - reverted

Reply:
actually - it's from your previous comment 
https://github.com/pandas-dev/pandas/pull/38416/files#r549454789